test/stdlib/UnsafeRawBufferPointer.swift

```diff
@@ -421,6 +421,21 @@ UnsafeRawBufferPointerTestSuite.test("copyMemory.overflow") {
       from: UnsafeRawBufferPointer(buffer))
 }
 
+// Use copyBytes without contiguouseStorage
```

Comment:
This is a good idea for a test, but it is passing in an `ArraySlice`, which still supports contiguous storage.